### DISCUSSION TITLE
ePO data for testing interoperability with OCDS

### DISCUSSION THREAD

**Discussion by ColinMaudry** [eprocurementontology/eprocurementontology#615] [Q&A]
Hello,

I work with the [Open Contracting Partnership](https://open-contracting.org) in order to study ways to make the ePO and the [Open Contracting Data Standard](https://standard.open-contracting.org/) (JSON schema) interoperable. More specifically, paving the way to an ePO => OCDS transformation.

In the process, I would like to be able to test certain assumptions with RDF data modeled according to the ePO. Any serialization would do, I plan to do SPARQL queries.

Is there ePO data publicly accessible? or sample/fictional data?

Many thanks 🙂 
cc @jpmckinney @yolile

**Comment by cristianvasquez** [comment]
Hello,

There are some Contract award Notices published between September and December 2023, accessible from this endpoint: https://publications.europa.eu/webapi/rdf/sparql

Currently, the easiest way to retrieve the graph for a notice y is by doing a [CONSTRUCT](https://api.triplydb.com/s/ax3SScOhx) over the corresponding `named-graph`.

There are some example queries that can be found [here](https://docs.ted.europa.eu/SWS/sample_app/sparql_queries.html) and [here](https://github.com/OP-TED/ted-rdf-docs/blob/main/notebooks/cpv-and-notices.md)

I hope this helps

Cristian

**Reply by ColinMaudry** [reply]
Thank you very much @cristianvasquez 

Why is so little data published? Do you know when data will be published using more ePO modules?

**Reply by ColinMaudry** [reply]
I observe this data uses ePO 3.1.0 (identifiers use `epo:identifier` predicate). Do you know if and when data using ePO 4.0 will be published?

**Comment by muricna** [comment]
Hello,
The data provided covers transformations of notices available on TED in TED-XML format from September to December.
Since January notices are published in TED in eForms format, we are currently working on the mapping of these notices using v.4.0.0 of the ontology they should be available later in the year.
Hope this provides you the information you require,
Natalie
